mutatex/core.py

```diff
@@ -911,10 +913,15 @@ def run(self):
             return False
 
         elif returncode == 0:
-            log.info("run %s has completed successfully" % self.name)
-            self.finished = True
-            self.process_output(**self.output_processing)
-            return True
+            i = self.check_values(**self.values_check)
+            if i == True:
+                log.info("run %s was terminated before completing due to unrecognized molecules" % self.name)
```

Comment:
here the run was not really terminated; it completed but unrecognised molecules were found by FoldX